flink-streaming-scala/src/main/scala/org/apache/flink/streaming/api/scala/StreamExecutionEnvironment.scala

```diff
@@ -690,12 +690,11 @@ class StreamExecutionEnvironment(javaEnv: JavaEnv) {
     f
   }
 
-
   /**
     * Registers a file at the distributed cache under the given name. The file will be accessible
     * from any user-defined function in the (distributed) runtime under a local path. Files
-    * may be local files (as long as all relevant workers have access to it), or files in a
-    * distributed file system. The runtime will copy the files temporarily to a local cache,
+    * may be local files (will be distributed via BlobServer), or files in a distributed file
```

Comment:
add `which` before `will` for better flow`. Also applies to other instances of this method.